addons/web/static/src/views/calendar/hooks.js

```diff
@@ -114,13 +115,20 @@ export function useFullCalendar(refName, params) {
         }
     });
 
-    let isWeekendVisible = params.isWeekendVisible;
+    let isWeekendVisible = params.weekends;
```

Comment:
you're right that the previous naming was terrible, but you can't change it in stable. In master though, I agree with that change

Reply:
Hello @aab-odoo , @mcm-odoo 
Previously we were always getting `undefined` in 'params.isWeekendVisible' and also 'isWeekendVisible' is not an appropriate option for fullcalendar, so I replaced it with an appropriate option for fullcalendar 'weekends' [docs](https://fullcalendar.io/docs/weekends) .
By this change I am able to achieve below PR specification -
```
Also when "Show Weekends" filter is in false state and we
refresh the page we are getting weekends in calendar view
of dashboard while the "Show Weekends" filter state remains
false.
```
Considering the above scenario kindly give your suggestions.

Thanks!!